src/auth/auth.service.ts

```diff
@@ -111,10 +119,10 @@ export class AuthService {
     return respond;
   }
 
-  async userLogout(userId: ObjectId) {
-    const user = await this.userModel.findById(userId);
+  async userLogout(body) {
+    const user = await this.userModel.findById(body.userId);
     user.hashedRefreshToken &&
-      (await this.userModel.findByIdAndUpdate(userId, { hashedRefreshToken: null }));
+      (await this.userModel.findByIdAndUpdate(body.userId, { hashedRefreshToken: null }));
```

Comment:
A nitpick. Destructuring userId? It seems like the only variable you need is userId.

Reply:
Thanks for the suggestion. Done!